pkgs/build-support/rust/build-rust-crate/configure-crate.nix

```diff
@@ -180,7 +180,7 @@ in ''
          export CARGO_FEATURE_$feature=1
        done
 
-       target/build/${crateName}/build_script_build > target/build/${crateName}.opt
+       target/build/${crateName}/build_script_build > target/build/${crateName}.opt || (cat target/build/${crateName}.opt && exit 1)
```

Comment:
I can't find a similar example somewhere else, but could we add some context to that output? Something like: `echo "build.rs failure output:" && cat ...`?

It's useful to show this, but knowing where it came from would be even better. :+1: otherwise